java/ql/test/experimental/query-tests/security/CWE-347/Auth0NoVerifier.qlref

```diff
@@ -0,0 +1 @@
+experimental/Security/CWE/CWE-347/Auth0NoVerifier.ql
```

Comment:
```suggestion
query: experimental/Security/CWE/CWE-347/Auth0NoVerifier.ql
postprocess: TestUtilities/PrettyPrintModels.ql
```
Try making this change and re-generating the test results. It should remove the absolute model numbers, which is what's causing tests to fail at the moment.